engine/flink/model-util/src/main/scala/pl/touk/nussknacker/engine/flink/util/transformer/aggregate/transformers.scala

```diff
@@ -86,10 +88,70 @@ object transformers {
             } else {
               keyedStream
                 .window(TumblingEventTimeWindows.of(Time.milliseconds(windowLength.toMillis)))
-                .aggregate(new UnwrappingAggregateFunction(aggregator))
+                .aggregate(new UnwrappingAggregateFunction(aggregator, _.value))
             }
           ExplicitUidInOperatorsSupport.setUidIfNeed(explicitUidInStatefulOperators(ctx), ctx.nodeId)(statefulStream)
             .map(_.map(aggregator.alignToExpectedType(_, expectedType)))
         }))
 
+  def sessionWindowTransformer(keyBy: LazyParameter[CharSequence],
+                            aggregateBy: LazyParameter[AnyRef],
+                            aggregator: Aggregator,
+                            windowLength: Duration,
+                            endSessionCondition: LazyParameter[java.lang.Boolean],
+                            variableName: String,
+                            explicitUidInStatefulOperators: FlinkCustomNodeContext => Boolean
+                           )(implicit nodeId: NodeId): ContextTransformation =
+      ContextTransformation.definedBy(aggregator.toContextTransformation(variableName, aggregateBy))
```

Comment:
I wonder if it would be a good idea to extract context transformation logic to some method to make it easier to not only add aggregation result but also e.g. session start time